ui/raidboss/common_replacement.js

```diff
@@ -119,6 +119,7 @@ export const commonReplacement = {
       ko: '--점프--',
     },
     '--knockback--': {
+      de: '--Rückstoß--',
```

Comment:
Maybe an odd question, but here to capitalize first-letter is right?

Reply:
The last time, I wanted to remove a capital letter from a translation, I was told to leave it as it was written. So I just did a copy and paste.

Reply:
Ah, I see. Thanks!